go.mod

```diff
@@ -3,68 +3,63 @@ module github.com/kubernetes-csi/csi-lib-utils
 go 1.12
 
 require (
-	github.com/container-storage-interface/spec v1.2.0
-	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/container-storage-interface/spec v1.3.0
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
-	github.com/golang/protobuf v1.3.5
-	github.com/googleapis/gnostic v0.3.1 // indirect
+	github.com/golang/protobuf v1.4.2
 	github.com/onsi/ginkgo v1.12.0 // indirect
-	github.com/pkg/errors v0.9.1 // indirect
 	github.com/stretchr/testify v1.5.1
-	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
-	golang.org/x/sys v0.0.0-20200406155108-e3b113bbe6a4 // indirect
-	google.golang.org/genproto v0.0.0-20200406120821-33397c535dc2 // indirect
-	google.golang.org/grpc v1.28.0
-	k8s.io/api v0.18.0
+	golang.org/x/net v0.0.0-20200707034311-ab3426394381
+	google.golang.org/grpc v1.29.0
+	k8s.io/api v0.19.0
 	k8s.io/client-go v1.18.0
```

Comment:
Please bump also client-go.

Was that not done by https://github.com/kubernetes-csi/csi-lib-utils/blob/master/release-tools/go-get-kubernetes.sh ?

Reply:
@pohly done. Also removed CSI spec bump from this PR.  PTAL :+1:

Reply:
> Also removed CSI spec bump from this PR

Then please also adapt the PR and commit description + title.

Reply:
@pohly done. updated PR title and commit message. :+1: